pkgs/racket-doc/scribblings/guide/other-editors.scrbl

```diff
@@ -284,3 +284,39 @@ provides support for syntax highlighting and building for Sublime Text.
 
 The @hyperlink["https://marketplace.visualstudio.com/items?itemName=evzen-wybitul.magic-racket"]{Magic Racket}
 extension provides Racket support including REPL integration and syntax highlighting in Visual Studio Code.
+
+@section{racket-langserver}
+
+The @tt{racket-langserver} is a @hyperlink["http://langserver.org/"]{Language Server Protocol}
+implementation for Racket. This project seeks to use @hyperlink["https://github.com/racket/drracket"]{DrRacket's public APIs} 
+to provide functionality that mimics DrRacket's code tools as closely as possible.
+
+@subsection{Installation and usage}
+
+A Racket runtime is a prerequisite, so before using @tt{racket-langserver}, ensure that a Racket runtime 
+is installed. You can install an from the @hyperlink["https://download.racket-lang.org"]{official download page}
+or install one from your package manager.
+
+@subsection{Atom}
+
+You can use the @hyperlink["https://github.com/cfinegan/atom-ide-racket"]{atom-ide-racket} package. The language 
+server will be automatically installed when @tt{atom-ide-racket} installs.
```

Comment:
Since Atom was discontinued over a year ago, I'd be inclined not to mention it here.